__tests__/app.test.js

```diff
@@ -84,12 +84,13 @@ describe('/api/articles/:id', () => {
     })
 })
 describe('/api/articles', () => {
-    test('responds with an array of article objects with comment_count created from comments.js and body property removed', () => {
+    test('GET 200: responds with an array of article objects with comment_count created from comments.js and body property removed', () => {
```

Comment:
This test description is me giving me concern that you have not used PSQL appropriately for this end point.

Reply:
I know this isn't the pr you've submitted, and I can see that the query is fine. This description could be something as simple as "Responds with an array of article objects with the required properties"